example/rbac.yaml

```diff
@@ -93,7 +93,7 @@ rules:
       - ""
     resources:
       - secrets
-    # resourceNames: [ "ca", "shoot-access-prometheus" ] # TODO: Andrey: P1: How to write code so we can use name-based restriction?
+    # resourceNames: [ "ca", "shoot-access-gardener-custom-metrics" ] # TODO: Andrey: P1: How to write code so we can use name-based restriction?
```

Comment:
When deploying the component from `gardenlet` this will be a go struct so setting the name can most likely be handled in code.